here_for_change/municipalities/templates/municipalities/ward_detail.html

```diff
@@ -352,174 +353,34 @@ <h2 class="benfritz fvs-100 text-center text-2xl text-white md:text-3xl">
     </div>
   </div>
 </section>
-<!-- Summary section -->
-<section class="bg-neutral-100 pt-20">
-  <div
-    class="mx-auto max-w-screen-xl gap-7 px-4 md:flex md:px-8 lg:gap-12 xl:px-4"
-  >
-    <!-- Left side bar -->
-    <div
-      id="toc"
-      class="sticky top-0 z-20 bg-neutral-100 py-6 text-sm md:top-16 md:h-[446px] md:w-1/4 md:py-0 md:text-base"
-    >
-      <div
-        id="side-anime"
-        class="relative flex flex-row gap-2 overflow-x-auto border-[#9333ea00] md:flex-col md:gap-0 md:overflow-x-visible md:rounded-3xl md:border-4 md:border-solid md:p-2"
-      >
-        <p
-          id="side-anime-banner"
-          class="benfritz absolute top-[-10%] left-[-20%] hidden rounded-md bg-[#7f00ff] p-2 text-white opacity-0 md:block"
-        >
-          Choose a category to explore!
-        </p>
-        {% for key, value in content.ward.items %}
-          {% if value.toc.show == True  %}
-            {% if forloop.counter0 == 1 %}
-            <a href="#{{value.toc.title|lower}}" class="btn-active toc-btn block md:w-auto md:h-12 md:flex text-center items-center gap-5 p-1 md:p-4 rounded-lg mb-3 cursor-pointer transition-all duration-[400ms] ease-in-out">
-              <span>
-                <i class="{{value.toc.icon}}"></i>
-              </span>
-              <span class="font-medium">
-                {{value.toc.title}}
-              </span>
-            </a>
-            {% else %}
-            <a href="#{{value.toc.title|lower}}" class="btn-inactive toc-btn block md:w-auto md:h-12 md:flex text-center items-center gap-5 p-1 md:p-4 rounded-lg mb-3 cursor-pointer transition-all duration-[400ms] ease-in-out">
-              <span>
-                <i class="{{value.toc.icon}}"></i>
-              </span>
-              <span class="font-medium">
-                {{value.toc.title}}
-              </span>
-            </a>
-            {% endif %}
-          {% endif %}
-        {% endfor %}
-      </div> 
-    </div>
-    <!-- Right side  -->
-    <div id="summary" class="toc pt-5 md:w-3/4">
-      {% include "./toc_summary.html" %}
-    </div>
+<div class="hidden">
+  <div class="current_ward">
+    {
```

Comment:
Why are we embedding JSON data as an HTML node and not in a <script>? @cyber237 @Hazmid

Reply:
I was trying to avoid using <script> altogether and query selecting the hidden div was the way I was able to access the template data in map.js. Do you know how we can we use script and also access the data in the script from map.js?

Reply:
<script> is fine for injecting page specific data. <script> should be avoided for functions etc.